services/todo-legacy/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "noImplicitAny": false,
```

Comment:
I am recommending to enable `strict` mode, it catches a lot of errors that would otherwise go unnoticed. (noImplicitAny is one of the options enabled in the strict mode).